app/models/tock_project.rb

```diff
@@ -0,0 +1,18 @@
+class TockProject < ActiveRecord::Base
```

Comment:
making a quick first pass at this whole thing, then I'll circle back with names, but yes, something like `billable_entity`...